app/models/concerns/submittable.rb

```diff
@@ -16,4 +20,11 @@ def receiving_office_phone_number
   def receiving_office
     @receiving_office ||= OfficeRecipient.new(benefit_application: self)
   end
+
+  def close_pdf
+    if @pdf.present? && @pdf.respond_to?(:close)
```

Comment:
when would a pdf not respond to `close`?
when would a pdf not be present? Why would we even be calling close_pdf if that was true?

Reply:
if the PDF is not generated properly and we try to close it, this will throw an exception